lib/modules/manager/sbt/extract.ts

```diff
@@ -49,10 +51,17 @@ const scalaVersionMatch = q
   )
   .handler((ctx) => {
     if (ctx.scalaVersion) {
+      const version = get(mavenVersioning.id);
+
+      let packageName = 'org.scala-lang:scala-library';
+      if (version.getMajor(ctx.scalaVersion) === 3) {
```

Comment:
Need to check it's a valid version first

Reply:
If it doesn't throw, we're safe without validity check I think

Reply:
As far as I can tell its not throwing, get back `null` in the event it fails an `isVersion` check https://github.com/renovatebot/renovate/blob/main/lib/modules/versioning/maven/index.ts#LL75C5-L75C5 (which in itself is doing a bunch of regex checks https://github.com/renovatebot/renovate/blob/main/lib/modules/versioning/maven/compare.ts#L280)